bin/all_sky_search/pycbc_coinc_hdfinjfind

```diff
@@ -45,7 +45,11 @@ def xml_to_hdf(table, hdf_file, hdf_key, columns):
     """ Save xml columns as hdf columns, only float32 supported atm.
     """
     for col in columns:
-        key = os.path.join(hdf_key, col)
+        if ':' in col:
+            col, new_col = col.split(':')
```

Comment:
What are you catching here (also add as comment in code)? .... This became clear after seeing what happened below, but wasn't obvious from comments/documentation in this function.